wski18n/i18n_resources.go

```diff
@@ -15,449 +15,257 @@
  * limitations under the License.
  */
 
-// Code generated by go-bindata.
-// sources:
-// wski18n/resources/de_DE.all.json
-// wski18n/resources/en_US.all.json
-// wski18n/resources/es_ES.all.json
-// wski18n/resources/fr_FR.all.json
-// wski18n/resources/it_IT.all.json
-// wski18n/resources/ja_JA.all.json
-// wski18n/resources/ko_KR.all.json
-// wski18n/resources/pt_BR.all.json
-// wski18n/resources/zh_Hans.all.json
-// wski18n/resources/zh_Hant.all.json
-// DO NOT EDIT!
-
 package wski18n
 
 import (
     "bytes"
     "compress/gzip"
     "fmt"
     "io"
```

Comment:
I suspect this file is not correctly generated. Why are these imports removed?
Each time, this file is regenerated, only the some byte arrays change.

Reply:
I generated it based on your document wski18n/README.md.
```
go get -u github.com/jteeuwen/go-bindata/...
sudo apt-get install go-bindata
go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources;
```